nx/public/sl/components.css

```diff
@@ -152,14 +170,88 @@ button {
     outline-offset: 4px;
   }
 
+  &:not(:disabled) {
+    cursor: pointer;
+  }
+
+  &.negative {
+    background: var(--s2-red-900);
+    border-color: var(--s2-red-900);
+    outline-color: var(--s2-red-900);
+
+    &:not(:disabled):hover {
+      background: var(--s2-red-800);
+      border-color: var(--s2-red-800);
+    }
+  }
+
   &.primary {
     background: var(--s2-gray-800);
     border: 2px solid var(--s2-gray-800);
     color: #fff;
 
+    &:not(:disabled):hover {
+      background: var(--s2-gray-900);
+      border-color: var(--s2-gray-900);
+    }
+
     &.outline {
       background: transparent;
       color: var(--s2-gray-800);
+
+      &:not(:disabled):hover {
+        background: transparent;
+        border-color: var(--s2-gray-600);
+        color: var(--s2-gray-600);
+      }
     }
   }
+
+  &:disabled {
+    opacity: 0.6;
+  }
+
+  &:not(:disabled):hover {
+    background: var(--s2-blue-800);
+    border-color: var(--s2-blue-800);
+  }
+}
+
+.sl-dialog {
+  --dialog-font-size: 16px;
```

Comment:
I think we should have a `styles.css` variable for this, but I'm also not sure if we need it. We should get 16px for free from the body.

Reply:
Actually we set the default font size to 19px on the body in nexter.css. I moved the variable to the exp.css along with the specific styling for this dialog.